ui/src/organizations/components/Variables.tsx

```diff
@@ -93,19 +122,21 @@ export default class Variables extends PureComponent<Props, State> {
     )
   }
 
+  public handleOpenImportOverlay = (): void => {}
+
   private handleFilterChange = (e: ChangeEvent<HTMLInputElement>) => {
     const {value} = e.target
     this.setState({searchTerm: value})
   }
 
   private handleFilterBlur() {}
 
-  private handleOpenModal = (): void => {
-    this.setState({overlayState: OverlayState.Open})
+  private handleOpenCreateOverlay = (): void => {
+    this.setState({createOverlayState: OverlayState.Open})
```

Comment:
Wondering if it's worthing setting the import overlay to closed here

Reply:
I don't think there's currently any way for the user to open the import overlay if the create overlay is open